sentipy/sentipy.py

```diff
@@ -1,3 +1,10 @@
+"""The Sentiment Investor Python Client library.
+
+This provides the main Senipy object, through which you can authenticate yourself.
```

Comment:
typo `Senipy` -> `SentiPy`

Reply:
The fact you noticed this typo shows the high level of detail of your review, so thank you for reviewing this @ZeevoX!